apps/admin_panel/assets/src/omg-page-dashboard/index.js

```diff
@@ -131,16 +131,16 @@ export default withRouter(
               <IntroContainer>
                 <h5>
                   This is where you will see all stats and trend of transaction within current
```

Comment:
You can see all the statistics related to transactions within the current account here.